components/badges.css

```diff
@@ -0,0 +1,151 @@
+@import url(/main.css);
+
+.main-container {
+  display: flex;
+  gap: auto;
+}
+
+.sidebar {
+  height: 900px;
+  background-color: black;
+  padding: 10px;
+  width: 25%;
+  display: fixed;
+}
+
+.sidebar h2 {
+  margin-left: 25px;
+  display: flex;
+  text-align: left;
+  color: #ebe8e8;
+}
+
+li a {
+  font-size: 1.4rem;
+  color: white;
+  height: 3rem;
+  text-align: center;
+  margin-top: 18px;
+  display: flex;
+  margin-left: 30px;
+  text-decoration: none;
+}
+
+h5 {
+  margin: 15px;
+  font-size: large;
+}
+
+h2 {
+  font-size: large;
+}
+
+/* Badges Section */
+
+.badges {
+  margin: 15px;
+  padding: 10px;
+  font-family: "Space Grotesk", sans-serif;
+  background-color: white;
+  border-radius: 5px;
+}
+
+.bell-icon {
+  color: black;
+  padding: 1rem;
+  position: absolute;
+}
+.bell-badge {
+  background-color: rgb(44, 44, 131);
+  color: white;
+  padding: 2px;
+  font-size: x-small;
+  position: relative;
+  left: 30px;
+  top: 5px;
+  border-radius: 60%;
+}
+
+.cart-icon {
+  color: black;
+  padding: 1rem;
+  padding-left: 2rem;
+  position: absolute;
+}
+
+.cart-badge {
+  background-color: rgb(44, 44, 131);
```

Comment:
Better to use variable for this color as it is being used repeatedly.

Reply:
Ok, Noted. Thanks for pointing out.